Server/Server.Core/Server.Core.Static/ServerBuilder.hpp

```diff
@@ -7,26 +7,28 @@
 #include <string>
 
 #include "Server.hpp"
+#include "SettingsManager.hpp"
 
 namespace Server::Builder
 {
 /**
  * @brief Class builder of Server.
  * @details Class based on builder pattern for creation instances of @link Server @endlink.
  */
-class ServerBuilder 
+class ServerBuilder
 {
 public:
     ServerBuilder()  = default;
-    ~ServerBuilder() = default;
+    explicit ServerBuilder(const SettingsManager& settingsManager) : _settingsManager(settingsManager) {}
 
-    inline ServerBuilder& setValue(std::pair<std::string, std::string> keyValue)
-    {
-        _arguments.insert(keyValue);
-        return *this;
-    }
+    ~ServerBuilder() = default;
 
-    inline ServerBuilder& setValue(DataAccess::IRepositoryManager* repoManager)
+    /**
+    * @brief Sets up a specific RepositoryManager
+    * @details Allows to set a specific RepositoryManager
+    * @warning This method is to MockTest only! It may disappear in future.
+    */
+    inline ServerBuilder& SetMockRepo(DataAccess::IRepositoryManager* repoManager)
```

Comment:
This method should initialize any repositories, not specifically linked to a mock repository

Reply:
Done.